Source/Core/Core/IOS/ES/ES.cpp

```diff
@@ -341,12 +341,8 @@ void ES::DoState(PointerWrap& p)
     p.Do(entry.m_opened);
     p.Do(entry.m_title_id);
     p.Do(entry.m_content);
-    p.Do(entry.m_position);
+    p.Do(entry.m_fd);
```

Comment:
Is it safe to use this FD after loading a state? Do we auto-reopen them inside FS (or elsewhere) during a load?

Also: This probably means you should be incrementing the state version.

Reply:
Yes, it's safe because FS saves the entire state (FDs + reopening host files).

I've bumped the savestate version, thanks for catching this